sdks/python/setup.py

```diff
@@ -167,6 +166,7 @@ def get_version():
 
 REQUIRED_TEST_PACKAGES = [
     'freezegun>=0.3.12',
+    'mock>=1.0.1',
```

Comment:
```suggestion
    'mock>=1.0.1<3.0.0',
```

Let's keep the upper bound for now to minimize the diff. Just making it a test dependency should address the TF 
and airflow issues. Updating the dep is tracked separately in BEAM-7369.